sirepo/template/radia.py

```diff
@@ -251,6 +261,23 @@ def post_execution_processing(
     return template_common.parse_mpi_log(run_dir)
 
 
+def stateful_compute_build_shape_points(data):
+    import csv
+
+    pts = []
+    with open(
```

Comment:
why not pkio.read_text?

Reply:
`csv` puts things into rows and columns for you. I prefer it to `pkio.read_text()` for this specific purpose